bin/run-viper-unsupervised.R

```diff
@@ -50,7 +50,10 @@ print ("Running unsupervised VIPER inference...")
 vpres <- run.viper.unsupervised(exprs, regul, regulon_minsize)
 print ("Done!")
 print ("Writing result..")
-viper.result <- t(vpres)
+viper.result <- vpres #it was originally t(vpres). Alana un-transposed b/c TFs by samples is easier to work with downstream
```

Comment:
Similar to above, I'm not clear on why this was transposed, but maybe there is something useful about that that I'm missing.